test/services/lessons/addMaterialUnit.test.js

```diff
@@ -0,0 +1,38 @@
+const assert = require('assert');
+const chai = require('chai');
+const { BadRequest } = require('@feathersjs/errors');
+const { getLesson, validateParams } = require('../../../src/services/lesson/hooks/addMaterial.js');
```

Comment:
Codacy found an issue: ['getLesson' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4977295)